app.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="h-screen flex flex-col font-sans text-primary-text bg-primary-bg theme-neon">
+    <div class="h-screen flex flex-col font-sans text-primary-text bg-primary-bg theme-violet">
```

Comment:
just switching themes so we're not to used to one yet until we have our final colors